api/ranking/rankingRouter.js

```diff
@@ -97,8 +98,14 @@ router.get("/histogram", restricted, async (req, res) => {
 router.get("/winner", async (req, res) => {
   try {
     let Today = await getPrompt();
-    if (Today.voting || Today.topThree || Today.active || !Today)
-      return res.status(400).json({ error: "No winners declared yet" });
+    let yesterdaysWinner;
+    if (Today.voting || Today.topThree || Today.active || !Today) {
+      // OLD CODE >> return res.status(400).json({ error: "No winners declared yet" });
+
+      // return the result of our getYesterdayWinner function
+      yesterdaysWinner = getYesterdaysWinner();
+      return res.status(200).json(yesterdaysWinner);
```

Comment:
Might be beneficial to add a `tomorrow: 1` flag of sorts so the FE has a signal to display that this is yesterdays winner they're seeing